packages/overlay/src/vaadin-overlay-stack-mixin.js

```diff
@@ -13,13 +13,20 @@ const getAttachedInstances = () =>
     .filter((el) => el instanceof HTMLElement && el._hasOverlayStackMixin && !el.hasAttribute('closing'))
     .sort((a, b) => a.__zIndex - b.__zIndex || 0);
 
+/**
+ * Returns all attached overlay instances excluding notification container,
+ * which only needs to be in the stack for zIndex but not pointer-events.
+ * @private
+ */
+const getOverlayInstances = () => getAttachedInstances().filter((el) => el.$.overlay);
```

Comment:
How about renaming `getAttachedInstances` now?

```suggestion
const getOverlayInstances = () => getOverlayStackInstances().filter((el) => el.$.overlay);
```

Reply:
I'd prefer to avoid renaming code in the bugfix since the change would be easier to read this way.